src/codeu/chat/client/ClientMessage.java

```diff
@@ -107,15 +106,46 @@ public void addMessage(Uuid author, Uuid conversation, String body) {
     updateMessages(false);
   }
 
+  // Delete message, removes last message
+  public void deleteMessage() {
+    System.out.println("Entered ClientMessage.deleteMessage()");
```

Comment:
I don't think we need this line, because of how the rest of the application is formatted. Do you think it will help keep clarity if we have this line?